packages/extension-commerce-bm-seo/.npmignore

```diff
@@ -0,0 +1 @@
+dev/
```

Comment:
This is a good addition.. we should be sure to update the starter-extension so that it has this file already.